virtualmachine/aws/s3util.go

```diff
@@ -0,0 +1,49 @@
+package aws
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+func getSession(region string) (*session.Session, error) {
```

Comment:
Can we have a generic getSession method which returns session and then can be used to get ec2 and s3 service clients. 
For that, will having a separate utils.go common to all different services such as ec2 and s3, make sense?

Reply:
This is a generic function. We now have to change the ec2 part to use this.

Reply:
We should move it out of s3utils.go then -- and create a new file if such common file does not already exist.